application/views/signup_form.php

```diff
@@ -1,6 +1,6 @@
 <script type="text/javascript">
     var controller = "<?=site_url('user/signup')?>";
-    var directto = "<?=base_url()?>";
+    var directto = "<?=site_url('user/login')?>";
```

Comment:
The users are not supposed to login immediately after they signed up. The administrator will confirm the user after which the user can login.